### DISCUSSION TITLE
RxJS Website down

### DISCUSSION THREAD

**Discussion by bruceharris** [ReactiveX/rxjs#7274] [Report issues other than bug]
https://rxjs.dev/ seems to be down. Is this the appropriate forum to report this? thanks

**Comment by mgabeler-lee-6rs** [comment]
Looks like someone forgot to renew the domain / pay the bill.

HTTPS refuses connections, HTTP sends a redirect to http://domain-registrar.storage.googleapis.com/expired.html?rxjs.dev

**Comment by jakovljevic-mladen** [comment]
Thanks for reporting. We're working on it.

**Comment by carere** [comment]
Do we have an hour ? A day ? Any time ? where the website will be available again ??
Difficult to dev without docs 😅

**Reply by bruceharris** [reply]
In the meantime you can find docs in the comments in the source code. Examples:
https://github.com/ReactiveX/rxjs/blob/master/src/internal/operators/debounceTime.ts - replace file name for any operator function
https://github.com/ReactiveX/rxjs/blob/master/src/internal/observable/of.ts - replace file name for any creation function

**Reply by jakovljevic-mladen** [reply]
We don't have any estimates unfortunately. This project and infra around it is maintained only in the free time of core team members and contributors. Please understand.

@bruceharris, thanks, this is exactly what I wanted to suggest as a workaround.

**Comment by jakovljevic-mladen** [comment]
The issue is now fixed and the site should be available again, we're sorry for the inconvenience.

@bruceharris, thanks for reporting.

**Comment by Clashsoft** [comment]
It is down once again.

**Reply by jakovljevic-mladen** [reply]
Yes, we're sorry for the inconvenience. For now, please use https://rxjs-dev.firebaseapp.com until [DNS issue](https://github.com/ReactiveX/rxjs/issues/7551) with the original URL is solved.
